src/main/java/evelyn/command/Ui.java

```diff
@@ -0,0 +1,102 @@
+package evelyn.command;
+
+import evelyn.exception.InvalidInputException;
+import evelyn.exception.NoInputException;
+
+import java.time.DateTimeException;
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * Houses all the logic for Evelyn's UI
+ */
+
+public class Ui {
+    public static String horizontalLine = "-----------------------------------------";
+    private boolean isChatting;
+    private String text = null;
+    private Parser parser;
+    private Scanner scanner;
+
+    /**
+     * Constructor of a Ui object.
+     * @param parser Parser used for the chatbot.
```

Comment:
I noticed that there’s no blank line between the description and the @param tag in the Javadoc for the constructor, and in other classes as well. Maybe adding a line between would be helpful.